oga/frontend/src/containers/QuestionList/QuestionList.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+
+import Question from '../../components/Question/Question';
+
+import { NavLink } from 'react-router-dom';
+
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+
+import * as actionCreators from '../../store/actions/index';
+
+class QuestionList extends Component {
+  componentDidMount() {
+    this.props.onGetAll();
+    // if (!this.props.log_status) {
+    //   this.props.history.push('/login');
+    // }
+  }
+
+  clickTodoHandler = (ar) => {
+    this.props.history.push('/questions/' + ar.id);
+  }
```

Comment:
1. How about changing name from clickTodoHandler to clickQuestionHandler?
2. How about changing variable ar to qs?